### ISSUE TITLE
SteamVR Input option nitpicks

### ISSUE THREAD

**Issue by ZwipZwapZapony** [jrbudda/Vivecraft_114#40]
Considering the existence of the radial menu, I'd suggest making the global game menu and open/close inventory controls in the action manifest have "requirement" set to "suggested" rather than "mandatory".
The former will _prompt_ the user to bind those controls, but the latter will _force_ them to. Before the SteamVR Input update for Vivecraft, I simply stuffed both of those things into the radial menu, which worked fine for me.

Relating to the above, I don't remember binding the open game menu control to any button before SteamVR Input stuff, but if I recall correctly, I do remember being able to back out of menus using a button anyway, which I'd like to be able to do without binding the global open/close menu button.
So with that in mind, I'd also like to suggest having a "close menu" bind specific to the GUI inputs tab, to allow backing out of menus with a button without having said button also open a menu.

~

TL;DR: Make the open/close game menu/inventory inputs "suggested", not "mandatory", and maybe also add a GUI-specific close menu input.
